include/lldb/Target/Language.h

```diff
@@ -231,6 +233,10 @@ class Language : public PluginInterface {
   static void GetDefaultExceptionResolverDescription(bool catch_on,
                                                      bool throw_on, Stream &s);
 
+  // SWIFT_ENABLE_TENSORFLOW
+  virtual CompletionResponse CompleteCode(Target &target,
+                                          std::string entered_code);
```

Comment:
Should this be `const std::string&` instead of `std::string`? If so, there are also other places that take a std::string argument instead of `const std::string&`

Reply:
Done!